dataset/dictionary/living-beings.json5

```diff
@@ -165,6 +165,53 @@
   },
 
 
+  //
+  // Liyue
+  //
+  {
+    en: "Redbill Pelican",
+    ja: "赤喙ペリカン",
+    zhCN: "朱喙鹈鹕",
+    pronunciationJa: "あかくちばしぺりかん",
+    tags: [ "living-being", "liyue" ],
+  },
+  {
+    en: "Fluff-Fleece Goat",
+    ja: "フワフワヤギ",
+    zhCN: "柔柔羊",
+    pronunciationJa: "ふわふわやぎ",
+    tags: [ "living-being", "liyue" ],
+  },
+  {
+    en: "Jadestone Turtle",
+    ja: "玉璜ガメ",
+    zhCN: "玉璜龟",
+    pronunciationJa: "ぎょくごうがめ",
+    tags: [ "living-being", "liyue" ],
+  },
+  {
+    en: "Venerable Jadestone Turtle",
+    ja: "玉璜古ガメ",
+    zhCN: "玉璜古龟",
+    pronunciationJa: "ぎょくごうこがめ",
+    tags: [ "living-being", "liyue" ],
+    notes: "遺瓏埠北部の湾内に生息する巨大亀",
+    notesZh: "一只生活在遗珑埠以北海湾的巨龟",
+  },
+  {
+    en: "Jadestone Turtle",
```

Comment:
Maybe you forget to update this to correct English translate when you copied-and-pasted.

Reply:
Sorry. Duplicate English: Jadestone Turtle
Plogrem
```
    en: "Jadestone Turtle",
    ja: "秋羽鴨",
    zhCN: "秋绒鸭",
```
after
https://genshin-impact.fandom.com/wiki/Velvetfall_Duck
```
    en: "Velvetfall Duck",
    ja: "秋羽鴨",
    zhCN: "秋绒鸭",
```